number_of_axis: 4
type_of_axis:
    - 1
    - 0
    - 0
    - 0
joint:
    omega:
        - 5.8119
        - 4.6600
        - 5.8119
        - 7.8540
    alpha:
        - 11.6238
        - 9.3200
        - 16.6054
        - 22.4400

#END
